src/ClockPanel.tsx

```diff
@@ -53,6 +54,17 @@ export function ClockPanel(props: Props) {
     return;
   }, [props.options.refresh, timezoneToUse]);
 
+  //refresh the time
+  let [time, description, err]: [moment.Moment, string, string | undefined] = useMemo(() => {
```

Comment:
I was rather confused with this variable and the `getTime` function. I now understand this `time` is the "targetTime". The time the counter will go up or down. Perhaps you should call it as such.

Also it is ok if you want to get the `description` from this same function but you can do two things:

* move this function out of the ComputeTime.tsx component. Since that is a react component and there's no reason to have it there. move it to its own function and perhaps you can rename it to something such as `calculateClockOptions` or `getProcessedClockOptions`. something that tells this function is doing calculations based on the options it gets.